src/DB/projects.json

```diff
@@ -691,20 +691,20 @@
     ]
   },
   {
-    "github_username": "vaibhav-xt",
+    "github_username": "itsme-shivamkumar",
     "Social_media": {
-      "gitHub": "",
-      "LinkedIn": "",
-      "Instagram": "",
+      "gitHub": "https://github.com/itsme-shivamkumar",
+      "LinkedIn": "https://www.linkedin.com/in/shivam-kumar-9575a7227/",
+      "Instagram": "https://www.instagram.com/sherlock_shivam/",
       "YouTube": "",
       "Twitter": ""
     },
     "Projects": [
       {
-        "link": "https://vaibhav-maurya.netlify.app/",
-        "title": "Portfolio site",
-        "description": "A portfolio project showcases an individual's or a professional's work, skills, and accomplishments in a concise and visually appealing manner to demonstrate their expertise and attract potential opportunities.",
-        "tech": ["CSS", "JavaScript", "HTML"]
+        "link": "https://github.com/itsme-shivamkumar/desume",
+        "title": "Desume | Resume Builder For Developers",
+        "description": "Desume Editor is a React-based web app that allows developers to edit their resumes with ease. With Desume Editor, developers can create or modify resumes with HTML and CSS directly in the app.",
+        "tech": ["React", "Firebase", "JavaScript", "CSS", "HTML", "Framer Motion"]
```

Comment:
you've deleted someone's contribution. Don't do this. 

You can close this PR & add create one

Reply:
I didn't knew it that I couldn't delete that, Sure, I'll close this and add new one